.pre-commit-config.yaml

```diff
@@ -41,6 +41,6 @@ repos:
         args: [requirements.in, -o, requirements.txt]
         files: ^requirements\.(in|txt)$
 -   repo: https://github.com/Riverside-Healthcare/djLint
-    rev: v1.25.0
+    rev: v1.29.0
```

Comment:
Request: Please change this back; the changed rulesets could introduce unintended consequences. This should be a separate PR.

Reply:
Can't we do something to prevent this from auto updating the number when setting up the devenv?

Reply:
Sorry my bad, I should have take extra care before commit. To be honest I do not know how did this changed.

Reply:
This automatically updates when you run the setup dev script (the pre commit hook get installed on your machine) - was mainly a question to @matmair if he is aware of any solution to prevent it from changing the version if we only install the pre commit hook. Also happens several times to me.

Reply:
Not bumping the versions on the versions lead to problems with yanked releases in the past. I would expect a dev to check what they submit to git more easily then diagnose yanked releases.

Reply:
Isn't this more like a CI task than a task that should run on every dev machine? I personally think this is more annoying than helpful. Also didn't quite understand your point.